### PR TITLE
Updated license to Apache 2.0

### PR DESCRIPTION
**Pull request by fmalatino**
**Description**
Added Apache 2.0 license text

Fixes # 30
This PR addresses issue 30

**How Has This Been Tested?**
No changes to functionality of code

**Checklist:**
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] New check tests, if applicable, are included
- [ ] Targeted model if this changed was triggered by a model need/shortcoming

### CODE REVIEW

**Comment by bensonr** [comment]
Please change the license spec in the steup.py that was added as part of #28

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   Copyright 2022 Allen Institute for Artificial Intelligence (AI2)
```
Comment (FlorianDeconinck) [inline LICENSE.md]: This needs to change.

Comment (fmalatino) [reply LICENSE.md]: Just the copyright and reference to AI2 or more?

Comment (FlorianDeconinck) [reply LICENSE.md]: The copyright yeah

**Review by FlorianDeconinck — APPROVED** [review]
LGTM

**Review by bensonr — APPROVED** [review]

**Review by oelbert — APPROVED** [review]
I am not a lawyer but looks good to me

**Pull request merged**